cmake/modules/SwiftSource.cmake

```diff
@@ -518,9 +530,9 @@ function(_compile_swift_files
     add_custom_command_target(
         sibopt_dependency_target
         COMMAND
-          "${PYTHON_EXECUTABLE}" "${line_directive_tool}" "${source_files}" --
+          "${line_directive_tool}" "@${file_path}" --
```

Comment:
Don't think that this is right.  Why are you dropping the use of `${PYTHON_EXECUTABLE}`?